responses/de/HassGetState.yaml

```diff
@@ -61,3 +61,25 @@ responses:
 
       wie_viele: |
         {{ query.matched | length }}
+
+      wo: |
+        {% if state.state == "not_home" %}
+          {{ slots.name | capitalize }} ist nicht da
```

Comment:
IMHO this could be misleading. If a person is neither at home nor in a defined zone, I would not expect `... ist nicht da` but rather `... ist außerhalb unterwegs`, `... ist in einem nicht definierten Bereich.` or similar. `nicht da` suggests the answer is binary (at home or not at home). Might just be me though because from a logical POV this works out just fine.

Reply:
what do you think about just `... ist unterwegs` ?